src/Application/Projects/Commands/UpdateProject/UpdateProjectCommandValidator.cs

```diff
@@ -0,0 +1,36 @@
+﻿using CoduTeam.Application.Common.Interfaces;
+using CoduTeam.Application.Projects.Commands.UpdateProject;
+
+namespace CoduTeam.Application.Projects.Commands;
+
+public class UpdateProjectCommandValidator : AbstractValidator<UpdateProjectCommand>
+{
+    private readonly IApplicationDbContext _context;
+
+    public UpdateProjectCommandValidator(IApplicationDbContext context)
```

Comment:
You have duplicated validators, create some base one and inherit it here) 
DRY :)